### DISCUSSION TITLE
canva pro con github

### DISCUSSION THREAD

**Discussion by medstudentstorvergata** [community/community#50528] [GitHub Education] [answered]
Salve a tutti, volevo sapere in che modo fosse possibile creare un account canva pro gratuito grazie all'account studenti github

**Comment by SilasRodrigues19** [comment, marked as answer]
E aí @medstudentstorvergata 

O benefício do Canva Pro deixou de ser oferecido pelo GitHub Student Developer Pack a algum tempo, quem já havia resgatado consegue ter o benefício por 1 ano mas quem não resgatou não consegue mais.

Veja aqui uma resposta de um dos Mantenedores do GitHub sobre esse assunto:

https://github.com/community/community/discussions/47259
